shell/platform/tizen/key_event_handler.cc

```diff
@@ -18,9 +21,28 @@ namespace {
 constexpr char kBackKey[] = "XF86Back";
 constexpr char kExitKey[] = "XF86Exit";
 
+// Keys that should always be handled by the app first but not by the system.
+const std::vector<std::string> kBindableSystemKeys = {
+    "XF86Menu",           "XF86Back",        "XF86AudioPlay",
+    "XF86AudioPause",     "XF86AudioStop",   "XF86AudioNext",
+    "XF86AudioPrev",      "XF86AudioRewind", "XF86AudioForward",
+    "XF86AudioPlayPause", "XF86AudioRecord", "XF86LowerChannel",
+    "XF86RaiseChannel",   "XF86ChannelList", "XF86PreviousChannel",
+    "XF86SysMenu",        "XF86SimpleMenu",  "XF86History",
+    "XF86Favorites",      "XF86Info",        "XF86Red",
+    "XF86Green",          "XF86Yellow",      "XF86Blue",
+    "XF86Subtitle",       "XF86PlayBack",    "XF86ChannelGuide",
+    "XF86Caption",        "XF86Exit",
+};
+
 }  // namespace
 
 KeyEventHandler::KeyEventHandler(FlutterTizenEngine* engine) : engine_(engine) {
+  if (!engine->renderer() || !engine->renderer()->IsValid()) {
+    return;
+  }
+  engine->renderer()->BindKeys(kBindableSystemKeys);
```

Comment:
Q. Does the flutter app now have the right to handle all these keys first? what is this decision based on?

Reply:
Yes, the app has the highest priority to handle key events that are specified by the list. https://github.com/flutter-tizen/flutter-tizen/issues/319#issuecomment-1016129187 explains the background on it.